### PR TITLE
initial jstat functionality

### PR DESCRIPTION
**Pull request by jedmpaz**
table isn't the greatest looking but it contains some simple statistics

### CODE REVIEW

File: static/customChart.html [hunk 1]
```diff
@@ -394,7 +455,8 @@ <h1 class="page-header">Custom Graphs</h1>
                            var ratio = getBarInfo(string1[i], type);
                            arr1[j] = ratio;
                            j++;
-                       };
+                        }
+                        stats(arr,arr1);
                        //mybarChart.destroy();
```
Comment (jedmpaz) [inline static/customChart.html]: This is what I got it to produce:
https://i.gyazo.com/5d24857a478654cd630b70a62ec24360.png

That's really weird, I'll be looking into it. Also I can just remove all the commented stuff out as well. I guess it's left over stuff from the old graph code.

**Comment by BenTheElder** [comment]
Actually don't worry about those if we don't need them. I've got a major
overhaul of the frontend to a single page app nearly done. Once this is
good to go I'll deal with that as part of the rewrite, unless you actually
want to get them now.

We should probably have better default field values for cases like this
though.

**Comment by jedmpaz** [comment]
Okay, honestly it won't take much to reimplement on the SPA, so I can just wait for that

**Comment by BenTheElder** [comment]
Ok cool. I'll try to get it up tonight, by the end of the weekend at the latest. I have it minimally working but it needs to be cleaned up and tested a bit more first.

**Comment by BenTheElder** [comment]
The aforementioned changes are up at #58.

**Comment by jedmpaz** [comment]
Updated to work with SPA. Currently has basic stats stuff for single data graphs, and correlation coefficient and covariance functionality for comparison bar charts.

**Comment by BenTheElder** [comment]
The merge of the latest changes looks botched (perhaps rebase instead? it will also make it easier to see which commits are actually new changes), and the new commits appear to all have the same message. Additionally, it only appears to only work for a single graph on the custom graphs page (the stats are overwritten by new graphs), but this looks pretty promising.

**Comment by jedmpaz** [comment]
Dang, it looked like I didn't have any problems when I merged, but yeah I'll rebase.

Also I've been playing around with multiple tables, and each separate table created bunches up towards the top instead of sticking with its corresponding graph. I'm looking into a way to fix that (still trying to understand the highcharts code a bit better), so once that gets done I'll update this again.

**Comment by BenTheElder** [comment]
Check out insertNewGraphRow for how the graph divs are inserted.

https://github.com/ChelsiAlise/workingdogdatadash/blob/181e8981ea00f77f97444648f37b45b837ae3872/static/index.js#L986

**Comment by jedmpaz** [comment]
Okay after me screwing up like twice I'm _pretty_ sure it's good to go. Lemme know if there's anything that passed under my radar.

**Comment by BenTheElder** [comment]
I think it might be easier to move a copy of the actual jstat work onto a fresh copy of master than try to completely find the changes that were reintroduced (such as bringing back customChart.html, which is no longer a page upstream).

**Comment by jedmpaz** [comment]
Oh what the heck I didn't see that. Yeah I'll do a fresh copy. I'm assuming this pull will just be closed then?

**Comment by BenTheElder** [comment]
That's easiest.

Though it is possible to continue with this pull.
Basically you copy this branch to another so you don't lose it. Then you
reset this one to master, add back your changes, and force push it.

Considering the prior conversation wouldn't make sense though, I'd just go
to a new branch and PR.

**Pull request closed without merging**